src/components/buttons/СategoryButton/CategoryButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { TouchableOpacity, Text } from 'react-native';
+import { useStyles } from 'react-native-unistyles';
+import { SvgProps } from 'react-native-svg';
+import { useTranslation } from 'react-i18next';
+
+import { styleSheet } from './CategoryButton.style';
+import { Categorys } from './CategoryButton.constans';
+import { sizes } from '@src/styling/sizes';
+
+type CategoryButtonProps = {
+	category: Categorys;
+	SvgIcon: React.FC<SvgProps>;
+	onPress: (category: Categorys) => void;
+};
+
+export const CategoryButton = ({ category, SvgIcon, onPress }: CategoryButtonProps) => {
+	const [isActive, setIsActive] = useState<boolean>(false);
```

Comment:
This is a state duplication. CategoryButton should not keep it's own "isActive' state, but rather receive it as Prop. We can safely assume that parent component of `CategoryButton` will have some array-like state of `selectedButtons` so the parent should decide if this button is active or not